lib/engine/game/base.rb

```diff
@@ -251,9 +251,17 @@ def purchasable_companies
       private
 
       def init_bank
+        return Bank.new(self.class::BANK_CASH[players.size], log: @log) if self.class::BANK_CASH.is_a?(Hash)
```

Comment:
it might just be cleaner to do 

cash = self.class::BANK_CASH
cash = cash[players.size] if cash.is_a?(Hash)